dnsaas/admin.py

```diff
@@ -0,0 +1,258 @@
+from django.contrib import admin
+from django.contrib.admin import SimpleListFilter
+from django.contrib.admin.sites import AdminSite
+from django.contrib.admin.widgets import AdminRadioSelect
+from django.contrib.contenttypes.models import ContentType
+from django.db import models
+from django.forms import (
+    NullBooleanSelect,
+    ModelForm,
+    ValidationError,
+)
+from powerdns.models.requests import (
+    DeleteRequest,
+    DomainRequest,
+    RecordRequest,
+)
+from django.utils.translation import ugettext_lazy as _
+from django_extensions.admin import ForeignKeyAutocompleteAdmin
+
+from powerdns.models.powerdns import (
+    CryptoKey,
+    Domain,
+    DomainMetadata,
+    Record,
+    SuperMaster,
+)
+from powerdns.models.templates import (
+    DomainTemplate,
+    RecordTemplate,
+)
+from powerdns.models.tsigkeys import TsigKey
+from powerdns.utils import DomainForRecordValidator
+
+
+admin_site2 = AdminSite('admin2')
+
+
+RECORD_LIST_FIELDS = (
+    'name',
+    'type',
+    'content',
+    'ttl',
+    'prio',
+)
+
+
+class ReverseDomainListFilter(SimpleListFilter):
+    title = _('domain class')
+
+    # Parameter for the filter that will be used in the URL query.
+    parameter_name = 'domain_class'
+
+    def lookups(self, request, model_admin):
+        return (
+            ('fwd', _('domain:forward')),
+            ('rev', _('domain:reverse')),
+        )
+
+    def queryset(self, request, queryset):
+        q = (
+            models.Q(name__endswith='.in-addr.arpa') |
+            models.Q(name__endswith='.ip6.arpa')
+        )
+        if self.value() == 'fwd':
+            return queryset.exclude(q)
+        if self.value() == 'rev':
+            return queryset.filter(q)
+
+
+_domain_filters = (
```

Comment:
do we need to extract it to some variable? can't we define it directly in domain admin?